care/facility/models/daily_round.py

```diff
@@ -504,18 +505,29 @@ def set_push_score(item):
 
     def save(self, *args, **kwargs):
         # Calculate all automated columns and populate them
-        self.glasgow_total_calculated = (
-            self.cztn(self.glasgow_eye_open)
-            + self.cztn(self.glasgow_motor_response)
-            + self.cztn(self.glasgow_verbal_response)
-        )
-        self.total_intake_calculated = sum([x["quantity"] for x in self.infusions])
-        self.total_intake_calculated += sum([x["quantity"] for x in self.iv_fluids])
-        self.total_intake_calculated += sum([x["quantity"] for x in self.feeds])
-
-        self.total_output_calculated = sum([x["quantity"] for x in self.output])
-
-        # self.pressure_sore = self.update_pressure_sore()
+        if (
+            self.glasgow_eye_open is None
+            and self.glasgow_motor_response is None
+            and self.glasgow_verbal_response is None
```

Comment:
Are these fields always compounded?
is there any case for only some of the fields used to calculate the score?